themes/default/layouts/partials/blog/search.html

```diff
@@ -1,58 +1,3 @@
-<script type="text/javascript">
-    (function (w, d, t, u, n, s, e) {
-        w["SwiftypeObject"] = n;
-        w[n] =
-            w[n] ||
-            function () {
-                (w[n].q = w[n].q || []).push(arguments);
-            };
-        s = d.createElement(t);
-        e = d.getElementsByTagName(t)[0];
-        s.async = 1;
-        s.src = u;
-        e.parentNode.insertBefore(s, e);
-    })(window, document, "script", "//s.swiftypecdn.com/install/v2/st.js", "_st");
-
-    _st("install", "xxxQB1LC9DE212h8eJmh", "2.0.0");
-</script>
-
-<div id="search-container">
-    <div id="search-form">
-        <form class="relative flex">
-            <i class="fas fa-search absolute inset-y-0 flex items-center pl-3 text-gray-500"></i>
-            <input
-                id="search-query"
-                type="text"
-                class="st-default-search-input py-2 px-4 pl-8 text-sm border border-gray-400 focus:border-blue-500 focus:outline-none rounded transition"
-                placeholder="Search the blog"
-            />
-        </form>
-    </div>
-    <div id="search-results" class="relative"></div>
+<div>
+    <div id="search" data-app-id="{{ getenv "ALGOLIA_APP_ID" }}" data-search-key="{{ getenv "ALGOLIA_APP_SEARCH_KEY" }}" data-facets="Docs,Registry" data-index="production"></div>
```

Comment:
This should be `Blog`, not `Docs,Registry`:

```suggestion
    <div id="search" data-app-id="{{ getenv "ALGOLIA_APP_ID" }}" data-search-key="{{ getenv "ALGOLIA_APP_SEARCH_KEY" }}" data-facets="Blog" data-index="production"></div>
```